README.md

```diff
@@ -168,6 +126,7 @@ The required files mentioned above can be found here: https://github.com/cloudfo
 This process executes action scripts provided by services in restricted environment.
 More information on how to configure action scripts is documented here: https://github.com/cloudfoundry-incubator/service-fabrik-broker/wiki/Deployment-hooks-for-service-lifecycle-operations
 
+If predeployment actions are enabled in service catalog then deployment hook process has to be running.
```

Comment:
Can you add if I do not want any actions to be executed, what should be done?

Reply:
Added.